### PR TITLE
Add errors section to Amplitude Govern docs

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…acking-plan/publishing**](https://docs-cyw7n1hye.vercel.app/workspace/tracking-plan/publishing) |
|:-:|
|<a href="https://docs-cyw7n1hye.vercel.app/workspace/tracking-plan/publishing">Screenshot of /workspace/tracking-plan/publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/8qNitJ1_RtGQ8woGzb_skUx0c4g_.png">(view full size)</a> |







Commit 75edfb84d16337c4e696540f4c8381e10bd8ad64 (https://docs-cyw7n1hye.vercel.app).

File: pages/workspace/tracking-plan/publishing.mdx [hunk 1]
```diff
@@ -139,6 +141,27 @@ Avo will not impact any events or properties that are only defined in Govern, or
   style={{ margin: '32px 0 -32px 0' }}
 />
 
+### Govern Publishing Errors
```
Comment (hugihlynsson) [inline pages/workspace/tracking-plan/publishing.mdx:144]: The image above this title belongs to the webhook section below—all this new content should probably be above it.

File: pages/workspace/tracking-plan/publishing.mdx [hunk 2]
```diff
@@ -139,6 +141,27 @@ Avo will not impact any events or properties that are only defined in Govern, or
...
+
+The following errors might occur when publishing your tracking to Govern. When publishing errors occur the error message in the Avo UI will contain which items could not be updated along with the error codes.
+
+#### 404 Not Found
+
+We tried to update an event that doesn't exist in your Amplitud Govern Schema. If this error occurs for all of the events you are attempting to publish it might indicate that your Amplitude Govern Schema has not been initialized. Learn more about initializing your Amplitude Govern Schema in [the Amplitude Govern docs](https://help.amplitude.com/hc/en-us/articles/360047579451-Initialize-and-manage-the-Schema). If your Govern Schema has been initialized, and the error persists, please contact support.
```
[the remark this answers is not in the document]
Comment (logason) [reply pages/workspace/tracking-plan/publishing.mdx]: Fixed in 0e9b533!

File: pages/workspace/tracking-plan/publishing.mdx (hunk 1 above)
Comment (logason) [reply pages/workspace/tracking-plan/publishing.mdx:139]: Fixed in 0e9b533 ✨

**Pull request merged**